sqlalchemy_redshift/dialect.py

```diff
@@ -135,6 +136,32 @@ class RedshiftImpl(postgresql.PostgresqlImpl):
 ])
 
 
+class TIMESTAMPTZ(sa.dialects.postgresql.TIMESTAMP):
+
+    __visit_name__ = 'TIMESTAMPTZ'
+
+    def __init__(self):
+        super(TIMESTAMPTZ, self).__init__(timezone=True)
```

Comment:
Can you add a docstring here explaining why this is necessary? I assume TIMESTAMPTZ is a Redshift-specific datatype name?

Reply:
Added!